src/NuGet.Core/NuGet.Packaging/Rules/AnalysisResources.resx

```diff
@@ -164,6 +167,9 @@
   <data name="MisplacedTransformFileWarning" xml:space="preserve">
     <value>The transform file '{0}' is outside the 'content' folder and hence will not be transformed during installation of this package. Move it into the 'content' folder.</value>
   </data>
+  <data name="NoRefOrLibFolderInPackage" xml:space="preserve">
+    <value>This package does not contain a lib or ref folder, and is therefore being treated as compatible for ALL frameworks. If the libraries are contained in a build/&lt;tfm&gt; folder, consider creating the directory "lib/&lt;tfm&gt;/_._" within the package.</value>
```

Comment:
Like that...

Some other suggestions:

This package does not contain a lib/ or ref/ folder, and is therefore being treated as compatible for ALL
frameworks. Since framework specific files were found under the build/ directory for 'net461' and 'netstandard2.0', consider creating the following empty files to correctly narrow the compatibility of the package:
- lib/net461/_._
- lib/netstandard2.0/_._

Reply:
I think it should point to error specific docs page that tells you how to fix this.